src/org/etools/j1939_84/controllers/part01/Part01Step01Controller.java

```diff
@@ -93,34 +93,33 @@ public ResultsListener getResultsListener() {
 
         while (getDataRepository().getVehicleInformation() == null) {
             getDateTimeModule().pauseFor(500);
-            updateProgress("Step 1.1.e Collecting Vehicle Information");
+            updateProgress("Step 6.1.1.1.e Collecting Vehicle Information");
         }
         getListener().onVehicleInformationReceived(getDataRepository().getVehicleInformation());
     }
 
     @Override
     protected void run() throws Throwable {
-        incrementProgress("Step 1.1.a - c Displaying Warning Message");
-        String message = "Ready to begin Part 1" + NL;
-
+        incrementProgress("Step 6.1.1.1.a - Vehicle Data Collection");
         // 6.1.1.1.a. Confirm the vehicle is in a safe location and condition for the test.
-        message += "a. Confirm the vehicle is in a safe location and condition for the test" + NL;
+        String message = "a. Confirm the vehicle is in a safe location and condition for the test" + NL;
 
         // 6.1.1.1.b. Confirm that the vehicle battery is well charged. ([Battery voltage >> 12 V].)
-        message += "b. Confirm that the vehicle battery is well charged. (Battery voltage >> 12 volts)" + NL;
+        message += "b. Confirm the vehicle battery is well charged. (Battery voltage >> 12 volts)" + NL;
 
         // 6.1.1.1.c. Confirm the vehicle condition and operator control settings according to the engine manufacturer’s
         // instructions.
         message += "c. Confirm the vehicle condition and operator control settings according to the engine manufacturer’s instructions"
-                + NL;
+                + NL + NL;
+        message += "Please press OK to continue";
 
-        displayInstructionAndWait(message, "Start Part 1", WARNING);
+        displayInstructionAndWait(message, "Step 6.1.1.1.a, b & c", WARNING);
 
         // 6.1.1.1.d. Turn the ignition key to on.
-        incrementProgress("Step 1.1.d Ensuring Key ON/Engine OFF");
-        ensureKeyStateIs(KEY_ON_ENGINE_OFF);
+        incrementProgress("Step 6.1.1.1.d - Waiting for " + KEY_ON_ENGINE_OFF.name);
```

Comment:
Since we cleaning up stuff, we should move this `incrementProgress` into the `ensureKeyStateIs` method, so the formatting is the same in all places.